src/Symfony/Component/Security/Core/Authorization/AuthorizationChecker.php

```diff
@@ -54,10 +54,10 @@ public function __construct(TokenStorageInterface $tokenStorage, AuthenticationM
             $this->tokenStorage->setToken($token = $this->authenticationManager->authenticate($token));
         }
 
-        if (\is_array($attributes)) {
+        if (\is_array($attribute)) {
             throw new InvalidArgumentException(sprintf('Passing an array of Security attributes to %s() is not supported.', __METHOD__));
```

Comment:
Shouldn't we remove this check altogether? ie we shouldn't care about what is passed here, and always pass `[$attribute]` below. Wouldn't this make sense?